Add What's New section to Studio 7.10 (#257)
mulesoft/docs-studio: modules/ROOT/pages/whats-new-in-studio.adoc

= What’s New in Studio

> Anypoint Studio 7.10 introduces a notification mechanism to help you keep up to date with your modules versions in Exchange, and also enables you to propagate metadata to flows and sub-flows.

Reviewer: ```suggestion
Anypoint Studio 7.10 introduces a notification mechanism to help you keep your module versions in Exchange up to date. Studio 7.10 also enables you to propagate metadata to flows and subflows.
```

Reviewer: I separated these because the initial version could imply that the notification mechanism enables you to propagate data.

> == Modules Updates Notifications

Reviewer: ```suggestion
== Module Update Notifications
```
You need not make "module" or "update" plural, because they are adjectives (types) of "updates," which is plural.

> Every time Studio starts up it checks for updated modules in your open projects. You can choose to update to the latest minor version of your module or upgrade directly to the latest version available.

Reviewer: "start up" is what is called a _colocation_.  There is an entry for these in the style ref., but they are essentially two-word verbs created by including a preposition. When translated literally, they can cause problems, and they ofter don't make literal sense: why "up," for example. (I had students in Japan who were always asking me about these: Why do you "put on" but "take off" clothing? English is weird....)

So...can we replace "start up" with "initiates" or something similar?

Reviewer: ```suggestion
Every time Studio starts, it checks for updates to modules in your open projects. Based on what it finds, you can choose to update to the latest minor version of your module or upgrade directly to the latest version available.
```

Author: Great feedback. Thank you.
I'll review my other usages of colocations across my content. 👍

> You can also prompt Studio to check for modules updates:

Reviewer: ```suggestion
You can also manually prompt a running instance of Studio to check for module updates:
```

. Ensure that you are logged in to Anypoint Platform.

> . Select the *Manage Mule Modules* (image:manage-mule-modules-icon.png[2%,2%]) button.

Reviewer: ```suggestion
. Click the *Manage Mule Modules* (image:manage-mule-modules-icon.png[2%,2%]) button:
```

+
image::manage-mule-modules-location.png[]

[CAUTION]

> This feature does not support modules that are inherited from a parent pom file.

Reviewer: Why is this a Caution: is there a potential failure?

See xref:update-modules.adoc[Update Your Modules] for more information.

> == Automatic Refactor When Renaming a Flow

Reviewer: ```suggestion
== Automatic Refactoring Caused by Flow Renaming
```

> When you rename a flow in Anypoint Studio, your change is now automatically refactored across your project.

Reviewer: ```suggestion
Note that when you rename a flow in Anypoint Studio, your change is automatically refactored across your project.
```
Anything more to say about this? It's kind of sparse for a whole separate subtopic....

== Metadata Propagation Between Flows

> Studio can infer the metadata of your current flow so that, when you use a Flow Reference component to reference another flow, you can set the metadata from your current flow as custom metadata for your referenced flow. You can also extract an operation from your current flow into a new flow or sub-flow and Studio can automatically declare the metadata as custom metadata of a new flow.

Reviewer: Are these things automatic or does the audience have to configure them? I am guessing the former:
```suggestion
Studio infers the metadata of your current flow so that when you use a Flow Reference component to reference another flow, you can set the metadata from your current flow as custom metadata for your referenced flow. You can also extract an operation from your current flow into a new flow or subflow, and Studio automatically declares the metadata as custom metadata of a new flow.
```

See xref:metadata-propagation-between-flows.adoc[Metadata Propagation Between Flows] for more information.

== Dark Theme Experience Improvements

> Studio now automatically detects the theme of your OS and adjusts its light or dark theme accordingly by default.

Reviewer: "automatically" negates the need for "by default":
```suggestion
Studio now automatically detects the theme of your OS and adjusts its light or dark theme accordingly.
```

> To manually change from light to dark theme, select the *Switch to Dark Theme* image:dark-theme-icon.png[] button.

Reviewer: ```suggestion
To manually change from the default light theme to the dark theme, click the *Switch to Dark Theme* image:dark-theme-icon.png[] button:
```

image::enable-dark-theme.png[]

> To switch back to the light theme, in the top toolbar, select the *Switch to Light Theme* image:light-theme-icon.png[] button.

Reviewer: Do we have to include locator text when showing a screen capture? If not...
```suggestion
To switch back to the light theme, click the *Switch to Light Theme* image:light-theme-icon.png[] button:
```

image::enable-light-theme.png[]

> You must restart Studio for this change to fully take effect.

Reviewer: ```suggestion
Restart Studio to force this change to immediately take effect.
```
